charts/gardener/operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,177 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:gardener-operator
+  labels:
+    app: gardener
+    role: operator
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+rules:
+- apiGroups:
+  - operator.gardener.cloud
+  resources:
+  - gardens
+  - gardens/status
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
```

Comment:
I believe `patch` is missing, the event controller patches existing events to increase the event's `count`.